src/pages/Movie/Movie.module.scss

```diff
@@ -0,0 +1,164 @@
+$shadow-color: rgba(0, 0, 0, 0.9);
```

Comment:
create a variables for each color in this style module please,not only for shadow-color. Also, lets name it correcty right now, so, e.g. if u are using the black color, name it as $black, in ur case ur have to create a variable $black: rgb(0, 0, 0) and rename ur variable as $black-90: rgba($black, 0.9) (so the naming shows the 0.9 opacity of black). Do it for each color here (for the name use this https://chir.ag/projects/name-that-color, just paste a HEX code here and it will give u a name for the color. Store the values in rgb as it is rn)